### ISSUE TITLE
[RPM] cannot open /etc/td-agent-bit/plugins.conf file

### ISSUE THREAD

**Issue by pszemus** [fluent/fluent-bit#1089]
## Bug Report

**Describe the bug**
Fluent Bit is configured to include `plugins.conf` by default but RPM package does not provide one.

**To Reproduce**
- Example log message if applicable:
```
[Warning] [config] I cannot open /etc/td-agent-bit/plugins.conf file
```
- Steps to reproduce the problem:
1. install `td-agent-bin` from RPM package
2. run `/usr/local/bin/td-agent-bit -c /etc/td-agent-bit/td-agent-bit.conf` or start as a service

**Expected behavior**
File `plugins.conf` is not included by default or it is installed by RPM package

**Your Environment**
* Version used: 1.0.4
* Configuration: default, provided by RPM package
* Operating System and version: CentOS Linux release 7.4.1708 (Core)

**Comment by edsiper** [comment]
Fixed by 468221da

Future packages will be fixed. thanks.

**Comment by kaizen1** [comment]
This is still a problem with latest Bionic Ubuntu packages.
